pom.xml

```diff
@@ -1906,13 +1906,13 @@
         <axis2-java2wsdl-maven-plugin.version>1.6.3</axis2-java2wsdl-maven-plugin.version>
 
         <!-- Axis2 Version -->
-        <axis2.wso2.version>1.6.1-wso2v41</axis2.wso2.version>
+        <axis2.wso2.version>1.6.1-wso2v59</axis2.wso2.version>
         <axis2.osgi.version.range>[1.6.1, 1.7.0)</axis2.osgi.version.range>
 
         <!-- Apache Axiom -->
-        <orbit.version.axiom>1.2.11-wso2v16</orbit.version.axiom>
-        <axiom.wso2.version>1.2.11-wso2v16</axiom.wso2.version>
-        <exp.pkg.version.axiom>1.2.11-wso2v16</exp.pkg.version.axiom>
+        <orbit.version.axiom>1.2.11-wso2v18</orbit.version.axiom>
```

Comment:
axiom,axis2, will not be able to update since components come in from kernel

Reply:
@Arshardh  please send the same pr to product repo and merge that end.

Reply:
demoted them